test/unit/test_language_def_parser.py

```diff
@@ -0,0 +1,154 @@
+from pathlib import PurePosixPath
+
+import pytest
+
+from exasol_script_languages_container_tool.lib.models.language_activation import (
+    BuiltInLanguageDefinitionURL,
+    ChrootPath,
+    LanguageDefinitionURL,
+    SLCLanguage,
+    SLCParameter,
+    UdfClientAbsolutePath,
+    UdfClientRelativePath,
+)
+from exasol_script_languages_container_tool.lib.tasks.upload.language_def_parser import (
+    parse_language_definition,
+)
+
+CHROOT_PATH_PARAMETERS = [
+    (
+        "/defaultbfs/default/slc",
+        ChrootPath(
+            bucketfs_name="defaultbfs",
+            bucket_name="default",
+            path_in_bucket=PurePosixPath("slc"),
+        ),
+    ),
+    (
```

Comment:
This and the next element looks the same

Reply:
Done